gradle.properties

```diff
@@ -25,6 +25,9 @@ protobufVersion=3.20.3
 jacocoVersion=0.8.10
 ballerinaToOpenApiVersion=2.1.0
 swaggerCoreVersion=2.2.22
+javaJsonPathVersion=2.9.0
```

Comment:
I think we added these to fix the compiler failure of jsondata. Can we remove them and check? IIRC these are only needed if we have used them in the native code

Reply:
Yeah, we added these to check the jsondata compiler plugin issue and we can remove this.